packages/common/utils/src/array/isSubset.ts

```diff
@@ -0,0 +1,5 @@
+/** @tossdocs-ignore */
+
+export function isSubset(a: unknown[] = [], b: unknown[] = []): boolean {
+  return a.every(element => b.includes(element));
+}
```

Comment:
Could you tell me why you use default parameter?
this function has no JSDoc, and parameter's name is `a` and `b`
user has no idea what function it is, and must be check Docs.

<img width="620" alt="image" src="https://github.com/toss/slash/assets/57122180/9f5ab33c-ef99-4a38-91a9-27b3393e0b39">

> What can I do next ..? 🤔

I think it is more convenient to use parameter name such as `compared`, or `subSet` and `wholeSet` or `comparision` for user.

And Second suggest is deleting default parameter, make user to know who use typescript what is wrong, and check function's parameter or JSDoc

For example, my suggestion is below

<img width="591" alt="image" src="https://github.com/toss/slash/assets/57122180/a1c46653-0e40-454b-9f22-54edd80fd36a">

1. delete default parameter
 user know it's function has two arguments

<img width="511" alt="image" src="https://github.com/toss/slash/assets/57122180/423ec650-cd6e-44b4-a62f-0b6eafb9b070">

2. parameter's name is more readable

JSDoc is make more readable for user what this function it is.

<img width="664" alt="image" src="https://github.com/toss/slash/assets/57122180/396cfb5f-0248-421e-a5d7-a355724fa324">

Reply:
I appreciate your review!

It was my mistake to use default parameters, i will fix that.
and
Using JSDoc would make it easier for users to understand this function!

Thank you for the great idea 😃